otelcolbuilder/Makefile

```diff
@@ -51,9 +51,13 @@ GOEXPERIMENT=""
 LDFLAGS="-s -w"
 # We want to use boringcrypto for fips setup
 else
+ifeq ($(shell go env GOOS),windows)
+GOEXPERIMENT=cngcrypto
```

Comment:
Don't you want to keep debug symbols here just as we do for the BoringCrypto build?

Reply:
Good catch, I re-added them.